### PR TITLE
Document GitHub Local Actions integration

### PR DESCRIPTION
**Pull request by SanjulaGanepola**

### CODE REVIEW

**Comment by SanjulaGanepola** [comment]
@cplee This PR is to document the extension in the docs as we discussed

File: src/integrations.md [hunk 1]
```diff
@@ -5,3 +5,4 @@
 - [Gitea Actions runner](https://gitea.com/gitea/act_runner)
 - [github-act-runner](https://github.com/ChristopherHX/github-act-runner)
 - [Gradle Act plugin](https://github.com/pshevche/gradle-act-plugin)
+- [GitHub Local Actions](https://sanjulaganepola.github.io/github-local-actions-docs/)
```
Comment (cplee) [inline src/integrations.md]: Can you update the link to be clear that this is a VSCode extension?

File: src/introduction.md [hunk 2]
```diff
@@ -9,6 +9,13 @@ Run your [GitHub Actions](https://developer.github.com/actions/) locally! Why wo
 - **Fast Feedback** - Rather than having to commit/push every time you want to test out the changes you are making to your `.github/workflows/` files (or for any changes to embedded GitHub actions), you can use `act` to run the actions locally. The [environment variables](https://help.github.com/en/actions/configuring-and-managing-workflows/using-environment-variables#default-environment-variables) and [filesystem](https://help.github.com/en/actions/reference/virtual-environments-for-github-hosted-runners#filesystems-on-github-hosted-runners) are all configured to match what GitHub provides.
 - **Local Task Runner** - I love [make](<https://en.wikipedia.org/wiki/Make_(software)>). However, I also hate repeating myself. With `act`, you can use the GitHub Actions defined in your `.github/workflows/` to replace your `Makefile`!
 
+---
+
+> ✨ **Now Run GitHub Actions Locally Directly From VS Code!** ✨
```
Comment (ChristopherHX) [inline src/introduction.md]: I'm not a fan of emojis, only speaking for myself who can't merge this.

I think this fits better into the docs, your slogan reads more like advertising your product name of your extension than describing the benefit.
```suggestion
> ✨ **Now Manage and Run Act Directly From VS Code!** ✨  
```

_Casey has the last word about this, it's his website domain that hosts the content and I don't know his opinion_.  Eventually he would just take it as is.

**Review by ChristopherHX — COMMENTED** [review]
_It's silent here_

File: src/introduction.md (hunk 2 above)
Comment (cplee) [reply src/introduction.md]: Let's remove emojis for consistency with rest of docs.

**Comment by SanjulaGanepola** [comment]
@cplee I addressed the PR comments. I also opened a PR in the main repo to update the README as well if the change here is good: https://github.com/nektos/act/pull/2588

File: src/integrations.md [hunk 3]
```diff
@@ -5,3 +5,4 @@
 - [Gitea Actions runner](https://gitea.com/gitea/act_runner)
 - [github-act-runner](https://github.com/ChristopherHX/github-act-runner)
 - [Gradle Act plugin](https://github.com/pshevche/gradle-act-plugin)
+- [GitHub Local Actions](https://marketplace.visualstudio.com/items?itemName=SanjulaGanepola.github-local-actions)
```
Comment (cplee) [inline src/integrations.md]:
```suggestion
- [GitHub Local Actions for Visual Studio Code](https://marketplace.visualstudio.com/items?itemName=SanjulaGanepola.github-local-actions)
```

Comment (SanjulaGanepola) [reply src/integrations.md]: @cplee I see what you mean now. Can I go ahead with this change but keep it at the previous link I had? Or this link works as well.

Comment (cplee) [reply src/integrations.md]: Whatever link you want works for me - so long as the text of the link is clear what it is.

Comment (SanjulaGanepola) [reply src/integrations.md]: Great, I updated to the previous link. The PR to update the README uses the same link so it makes sense to keep it the same.

**Review by cplee — APPROVED** [review]

**Pull request merged**